src/tokens/TabWallet.tsx

```diff
@@ -1,231 +1,82 @@
-    // Transparency issue on Android present when a fragment is used - Nested Animated.View prevents it
-    <Animated.View>
-      <Animated.View
-        style={[styles.listHeaderContainer, animatedListHeaderStyles]}
-        onLayout={handleMeasureListHeaderHeight}
-      >
-        <View
-          style={[styles.nonStickyHeaderContainer]}
-          onLayout={handleMeasureNonStickyHeaderHeight}
-        >
-          <View style={styles.balanceSection}>
-            <AssetsTokenBalance showInfo={false} />
-          </View>
-          <ActionsCarousel key={'ActionsCarousel'} />
-        </View>
-        <AssetTabBar
-          activeTab={activeTab}
-          onChange={handleChangeActiveView}
-          displayPositions={displayPositions}
-        />
-      </Animated.View>
-
-      <AssetList
-        activeTab={activeTab}
-        listHeaderHeight={listHeaderHeight}
-        handleScroll={handleScroll}
-      />
-      {showClaimRewards && (
-        <Animated.View
-          style={[styles.footerContainer, animatedFooterStyles]}
-          onLayout={handleMeasureListFooterHeight}
-        >
-          <Button
-            type={BtnTypes.SECONDARY}
-            size={BtnSizes.FULL}
-            text={t('assets.claimRewards')}
+    <SafeAreaView style={styles.container} edges={['top']}>
+      <View style={styles.row}>
+        <Text style={styles.totalBalance} testID={'TotalTokenBalance'}>
+          {!hideWalletBalances && localCurrencySymbol}
+          {balanceDisplay}
+        </Text>
+        <HideBalanceButton hideBalance={hideWalletBalances} />
+      </View>
+      <ScrollView contentContainerStyle={styles.contentContainerStyle}>
+        {tokens.map((token, index) => (
+          <TokenBalanceItem
+            token={token}
+            key={index}
             onPress={() => {
-              AppAnalytics.track(AssetsEvents.tap_claim_rewards)
-              navigate(Screens.DappShortcutsRewards)
+              navigate(Screens.TokenDetails, { tokenId: token.tokenId })
+              AppAnalytics.track(AssetsEvents.tap_asset, {
+                ...getTokenAnalyticsProps(token),
+                title: token.symbol,
+                description: token.name,
+                assetType: 'token',
+              })
             }}
+            hideBalances={hideWalletBalances}
           />
-        </Animated.View>
-      )}
-    </Animated.View>
+        ))}
+      </ScrollView>
+    </SafeAreaView>
   )
 }
 
 const styles = StyleSheet.create({
-  listHeaderContainer: {
-    ...getShadowStyle(Shadow.SoftLight),
-    paddingBottom: Spacing.Regular16,
-    paddingHorizontal: Spacing.Regular16,
-    backgroundColor: Colors.white,
-    position: 'absolute',
-    width: '100%',
-    zIndex: 1,
+  container: {
+    flex: 1,
   },
-  nonStickyHeaderContainer: {
-    zIndex: 1,
-    paddingBottom: Spacing.Thick24,
+  row: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    gap: Spacing.Smallest8,
   },
```

Comment:
Small margin missing from top of amount row.
```suggestion
    marginTop: Spacing.Smallest8,
  },
```